control/control_loop.go

```diff
@@ -290,12 +290,12 @@ func (l *Loop) startBenchmark(loops int) error {
 // Stop stops then loop.
 func (l *Loop) Stop() {
 	if l.running {
+		l.running = false
```

Comment:
one of the errors I was seeing was "close of closed channel" and moving this line solved that issue so repeated calls to `Stop` don't attempt to close the ticker channel multiple times